Assignment/solution_student/index.html

```diff
@@ -1,35 +1,40 @@
+<!DOCTYPE HTML>
 
-Sample Content
-——-
+<html>
+<head>
+	<title>About me</title>
+	<meta <meta charset="utf-8">
+	<link rel="stylesheet" href="css/main.css">
+	<link href='https://fonts.googleapis.com/css?family=Open+Sans+Condensed:300,700' rel='stylesheet' type='text/css'>
+</head>
 
-ABOUT ME PAGE
+<body>
 
-I ENJOY LIFE AS A DEVELOPER
+	<nav class="main-nav">
+		<a href="index.html" id="selected-page">About me</a> |
+		<a href="resume/resume.html">Portfolio</a>
+	</nav>
 
-I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.
+	<h1 id="main-name">Joe Johnson</h1>
 
+	<hr>
 
-RESUME PAGE
+	<img id="profile-pic" src="images/avatar.png" alt="avatar image">
 
-Experience
-        
-I recently graduated from a Front-End Web Development course at General Assembly where I learned HTML, CSS, JavaScript, jQuery and how to be an awesome Front-End Web Developer! During my spare time, I enjoy senior water aerobics, pickling, and spending time with my goldfish, Jerry.
-    
-Relevant Experience
-    
-Web Developer, Relaxr 2015 - present
-Developed a multi-column layout blog, landing page, and contact forms that render on mobile devices.
+	<h3 id="whatareyou">I ENJOY LIFE AS A DEVELOPER</h3>
 
-Web Developer, Startup Matchmaker 2015 - present
-Used a design team's wireframes to develop this company's responsive homepage.
 
-Web Developer, CitiPix 2015 – present
-Built web app prototype that allows users to store and quickly retrieve photos using keywords.
+	<p id="detail1">I'm Joe Johnson, a Developer based in NYC. I Have ten years of experience in the graphic design world, specializing in the creation of responsive websites.</p>
 
+	<br>
```

Comment:
Avoid using the `<br>` tag for presentational purposes, in other words, if you're using it to space things out on the page, you should be using `padding` or `margin` in CSS instead